local/local.go

```diff
@@ -722,7 +722,15 @@ func untar(r io.Reader, dest string) error {
 			if err != nil {
 				return err
 			}
-			if _, err := io.Copy(fh, tr); err != nil {
+<<<<<<< HEAD
+			if _, err := io.CopyN(fh, tr); err != nil {
+=======
+			if _, err := io.CopyN(fh, tr, 1024) {
+				if err != nil {
```

Comment:
Could we move this to the previous line?

We'll also need to resolve the merge conflict :)

Reply:
Done

Reply:
thanks